src/main/java/org/maxgamer/quickshop/QuickShop.java

```diff
@@ -1852,7 +1852,7 @@ private void updateConfig(int selectedVersion) throws IOException {
             getConfig().set("config-version", ++selectedVersion);
         }
         if (selectedVersion == 133) {
-            getConfig().set("integration.griefprevention.delete-on-unclaim", false);
+            getConfig().set("integration.griefprevention.delete-on-claim-unclaimed", false);
```

Comment:
Never touch the older version upgrading code, it may break the upgrading.

The correct way is read the old key value in new upgrade block (in there is 138), then write new value with new key and remove the old key.

Reply:
Example:
```java

if(selectedVersion == 137){  // <- You should always open new upgrade block
boolean oldValue  = getConfig().getBoolean("integration.griefprevention.delete-on-unclaim", false);
getConfig().set("integration.griefprevention.delete-on-unclaim", null);
getConfig().set("integration.griefprevention.delete-on-claim-unclaimed", oldValue);
}
```

Reply:
The previous config options were restored to the old ones.
In the new config version, I did what you told, however in the griefprevention.create and griefprevention.trade, I needed to reset them, because some of the old values would be invalid in this new version.